doc/Extensions/Globe-Frontpage.md

```diff
@@ -12,6 +12,12 @@ $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google";//Only one available at present
 ```
 
+Location resolution happens as follows (iff `$config['geoloc']['latlng'] == true;`):
```

Comment:
Oh, I was not aware of that shortand.  Perhaps we should be more verbose for the documentation.